lib/src/sha512.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:convert';
 
 import 'digest.dart';
 import 'hash.dart';
+// ignore: uri_does_not_exist
 import 'sha512_fastsinks.dart' if (dart.library.js) 'sha512_slowsinks.dart';
 //import 'sha512_slowsinks.dart';
```

Comment:
[optional] If you'd like to clean this up while in this file anyway - a commented out import should have not been submitted with the last PR.

Reply:
done